python-frontend/src/main/java/org/sonar/python/types/v2/TypeUtils.java

```diff
@@ -31,4 +31,11 @@ static PythonType resolved(PythonType pythonType) {
     }
     return pythonType;
   }
+
+  public static PythonType ensureClassTypeWrapped(PythonType pythonType) {
```

Comment:
Note: I think this will further the ambiguity between the technical "wrapper" that is used to resolve `LazyType`, and the functional "wrapper" that is used to signify that we're dealing with an instance of a type rather than the raw type itself.
We probably want a method name that conveys the fact that we're just ensuring we have an `ObjectType`.